lib/house.rb

```diff
@@ -0,0 +1,45 @@
+
+class House
+  attr_reader :price,
+              :address,
+              :rooms,
+              :area
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    if @price.to_i > 500000
```

Comment:
This will never return true, because a string that does not start with an integer will be converted as '0'.